pvlib/temperature.py

```diff
@@ -650,19 +653,52 @@ def ross(poa_global, temp_air, noct):
 
     .. math::
 
-        T_{C} = T_{a} + \frac{NOCT - 20}{80} S
+        T_{C} = T_{a} + \frac{NOCT - 20}{80} S = T_{a} + k × S
 
     where :math:`S` is the plane of array irradiance in :math:`mW/{cm}^2`.
     This function expects irradiance in :math:`W/m^2`.
```

Comment:
Let others comment, but I think `Wm⁻²` looks cleaner than the larger font bolt+italic math type for units. Might be worth saving that for variables(?)
At least if sticking with `:math:` then I think superscript would be better than a `/`: ``` `:math:`mW{cm}^{-2} ```
```suggestion
    where :math:`S` is the plane of array irradiance in mWm⁻².
    This function expects irradiance in Wm⁻².
```